.github/workflows/main.yml

```diff
@@ -0,0 +1,69 @@
+name: "CI"
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+
+  Test:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    strategy:
+      matrix:
+        node_version: ['lts/*', node]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Install Node
+        uses: dcodeIO/setup-node-nvm@master
```

Comment:
Why not use the GitHub standard Node setup action?

Reply:
actions/setup-node does not support aliases like `lts/*` or `latest` https://github.com/actions/setup-node/issues/26

Reply:
I'd rather be explicit anyway and list out `[10, 12, 14]`.